### PR TITLE
Update docker docs for VL api

### PR DESCRIPTION
**Pull request by vody-am**
Docs update in relation to issue https://github.com/InternLM/lmdeploy/issues/1514 

## Motivation

Updating documentation around Docker images for new users.
It should be made clear to users what is expected, and a way to resolve issues described.

## Modification

Adding a warning about dependencies and an example of how to resolve by creating a new Docker image.

### CODE REVIEW

**Comment by lvhan028** [comment]
@vody-am If you are familiar with Chinese, could you please update the user guide in `docs/zh_cn/serving/api_server_vl.md`?

**Comment by vody-am** [comment]
@lvhan028 Google translate has been working well, so yes I will give it a shot :joy:

**Comment by lvhan028** [comment]
Yes, by setting `tp` parameter

**Comment by lvhan028** [comment]
Hi, @vody-am 
there are linting errors, which are able to fixed by running the following commands:
```
pip install pre-commit
cd the/root/path/of/lmdeploy
pre-commit install .
pre-commit run --all-files
```

**Review by AllentDan — APPROVED** [review]

**Review by lvhan028 — APPROVED** [review]

File: docs/en/serving/api_server_vl.md [hunk 1]
```diff
@@ -37,6 +37,27 @@ docker run --runtime nvidia --gpus all \
...
+
+WORKDIR /app
+
+RUN pip3 install --upgrade pip
+RUN pip3 install timm
+RUN pip3 install git+https://github.com/haotian-liu/LLaVA.git --no-deps
+
+COPY . .
+
+CMD ["lmdeploy", "serve", "api_server", "liuhaotian/llava-v1.6-34b"]
```
Comment (zhyncs) [inline docs/en/serving/api_server_vl.md:58]: In a real production environment, the deployed model path is usually stored on something like Open Storage Service. Almost never download and load from HuggingFace every time you deploy.

Comment (zhyncs) [reply docs/en/serving/api_server_vl.md:58]: Therefore, this is more of a simple reference and has little significance in actual production environments.

**Pull request merged**